templates/README.mustache

```diff
@@ -14,14 +19,12 @@
 {{#documentation.usage}}
 ## Usage
 
-{{/documentation.usage}}
 <!--consumer-badges start -->
 <!--consumer-badges end -->
-{{#documentation.usage}}
 
 {{{documentation.usage}}}
-{{/documentation.usage}}
 
+{{/documentation.usage}}
 {{#documentation.contributing}}
```

Comment:
does the same problem exist for the `contribution-badges` zone when `documentation.usage` and `documentation.contribting` are both empty?

Reply:
to be clear, i'm cool with merging this either way since i've actually seen impact from the piece you've resolved. however, if the same situation can exist for the other zone, it seems like it would at least be worth us capturing another issue

Reply:
i'm glad you caught this. i was planning to add this as well and lost track of that. i think the same thing will happen here.

Reply:
cool. i'd still be ok with it being a separate pr, though, if it is much effort to handle. i happened to try to scaffold the `.github` repo for `pragmatic-divops` last night and hit the `consumer-badges` zone issue there, so not holding that back for this could be valuable.